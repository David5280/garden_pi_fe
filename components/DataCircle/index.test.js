import React from 'react';
import { DataCircle } from './index';
import 'react-native';
import renderer from 'react-test-renderer';


test('renders correctly', () => {
  const tree = renderer.create(<DataCircle />).toJSON();
  expect(tree).toMatchSnapshot();
});