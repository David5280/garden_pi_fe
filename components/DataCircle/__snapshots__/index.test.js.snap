// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#d5fdd5",
      "borderColor": "#A14550",
      "borderRadius": 30,
      "borderWidth": 2,
      "height": 226.78000000000003,
      "justifyContent": "space-between",
      "marginBottom": 3,
      "marginLeft": 10,
      "marginRight": 10,
      "marginTop": 3,
      "padding": 10,
      "width": 300,
    }
  }
>
  <Text />
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        },
        Object {
          "backgroundColor": "#000",
          "borderRadius": 97.5,
          "height": 195,
          "width": 195,
        },
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "left": 0,
            "position": "absolute",
            "top": 0,
          },
          Object {
            "height": 195,
            "width": 97.5,
          },
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "borderBottomRightRadius": 0,
              "borderTopRightRadius": 0,
              "left": 0,
              "position": "absolute",
              "top": 0,
            },
            Object {
              "backgroundColor": "#228b22",
              "borderRadius": 97.5,
              "height": 195,
              "overflow": "hidden",
              "transform": Array [
                Object {
                  "translateX": 48.75,
                },
                Object {
                  "rotate": "NaNdeg",
                },
                Object {
                  "translateX": -48.75,
                },
              ],
              "width": 97.5,
            },
          ]
        }
      />
    </View>
    <View
      style={
        Array [
          Object {
            "left": 0,
            "position": "absolute",
            "top": 0,
          },
          Object {
            "height": 195,
            "width": 97.5,
          },
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "borderBottomRightRadius": 0,
              "borderTopRightRadius": 0,
              "left": 0,
              "position": "absolute",
              "top": 0,
            },
            Object {
              "backgroundColor": "#000",
              "borderRadius": 97.5,
              "height": 195,
              "overflow": "hidden",
              "transform": Array [
                Object {
                  "translateX": 48.75,
                },
                Object {
                  "rotate": "0deg",
                },
                Object {
                  "translateX": -48.75,
                },
              ],
              "width": 97.5,
            },
          ]
        }
      />
    </View>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "justifyContent": "center",
            "overflow": "hidden",
          },
          Object {
            "backgroundColor": "#fff",
            "borderRadius": 89.5,
            "height": 179,
            "width": 179,
          },
        ]
      }
    >
      <Text />
    </View>
  </View>
</View>
`;
